src/GovUk.Education.ExploreEducationStatistics.Common/Requests/QueryRequests.cs

```diff
@@ -0,0 +1,90 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using FluentValidation;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data.Query;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Requests;
+
+public record FullTableQueryRequest
+{
+    public Guid SubjectId { get; set; }
+
+    public List<Guid> LocationIds { get; set; } = new();
+
+    public TimePeriodQuery? TimePeriod { get; set; }
+
+    public IEnumerable<Guid> Filters { get; set; } = new List<Guid>();
+
+    public IEnumerable<Guid> Indicators { get; set; } = new List<Guid>();
+
+    public class Validator : AbstractValidator<FullTableQueryRequest>
+    {
+        public Validator()
+        {
+            RuleFor(context => context.LocationIds)
+                .NotEmpty();
+            RuleFor(context => context.TimePeriod)
+                .NotNull();
+
+            // NOTE: No rule for filters - a data set might have no filters!
```

Comment:
I don't think it's super necessary to explain the absence of something here.

Reply:
My thinking is that it's possible someone might see this code and ask themselves the question and wonder if it's been missed, and this comment prevents someone wasting any time exploring it as it answers the question immediately.